### DISCUSSION TITLE
How many images do I need in my custom dataset?

### DISCUSSION THREAD

**Discussion by lakshaydulani** [layumi/Person_reID_baseline_pytorch#359] [Q&A]
Hello all
I need to train the model on my personal dataset.

Assuming there are 5 different persons, how many images per person do I need?

**Comment by layumi** [comment]
Hi @lakshaydulani 

I just miss this. Yes. You may consider to fuse with Market-1501 or DuekMTMC-reID or other datasets together. 

Generally, I recommend 10,000 images are required to obtain a stable result.
